### PR TITLE
bep concept model new branch

### PR DESCRIPTION
**Pull request by Ninicorn**
the other concept model branch had merge conflicts, hence copied over to a new branch.

### CODE REVIEW

File: docs/source/concept_models/vivarium_gates_bep/concept_model.rst [hunk 1]
```diff
@@ -170,8 +170,8 @@ Here is a memo describing the rationale underlying the causal structure of this
      - same 
    * - b (∆oMMN)
      - MMN vs IFA on birthweight in g
-     - +51.24 (43.71 to 58.76) 
-     - fixed effects meta analysis of 13 trials from Keats 2019 :download:`memo <meta-analysis_MMN_vs_IFA_memo.docx>`
+     - +51.24 (43.71 to 58.76) fixed effects *or* +45.16 (32.31 to 58.02) random effects
```
Comment (aflaxman) [inline docs/source/concept_models/vivarium_gates_bep/concept_model.rst:173]: We decided to use the RE version in the sim, right?  Good to have both here, but also let's document clearly which one we have decided to use.

File: docs/source/concept_models/vivarium_gates_bep/concept_model.rst [hunk 2]
```diff
@@ -188,10 +188,16 @@ Here is a memo describing the rationale underlying the causal structure of this
      - TPP target
    * - x
      - crude birthweight in g shift between low and normal BMI women
-     - -160.44g (-196.65 to -124.23) fixed effect *or* -142.93g (-232.68 to -53.18) random effects
+     - -160.44g (-196.65 to -124.23) fixed effects *or* -142.93g (-232.68 to -53.18) random effects
```
Comment (aflaxman) [inline docs/source/concept_models/vivarium_gates_bep/concept_model.rst:191]: I thought it was "fixed effect" (singular) for meta-analysis, although I say Ali used the plural for the doc on the general method.  I will defer to the language she has selected after she double checks it in response to my prior comment

File: docs/source/concept_models/vivarium_gates_bep/concept_model.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/concept_models/vivarium_gates_bep/concept_model.rst:173]: yes, I made a note in the bottom of the table to use random effect(s).

File: docs/source/concept_models/vivarium_gates_bep/concept_model.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/concept_models/vivarium_gates_bep/concept_model.rst:191]: ok

**Pull request merged**